### ISSUE TITLE
Oppositional behavior

### ISSUE THREAD

**Issue by stefaniadv** [obophenotype/human-phenotype-ontology#11145]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@bcoleman96 - can we touch bases about this?

**Comment by pnrobinson** [comment]
Probably this wants to be a synonym of [Oppositional defiant disorder HP:0010865](https://hpo.jax.org/browse/term/HP:0010865)

**Comment by pnrobinson** [comment]
I think we need to relabel Oppositional defiant disorder HP:0010865

**Primary label:** Age-inappropriate oppositional behavior

**Def**: A pervasive and characteristic pattern of arguing, defiance, and hostility toward others, particularly authority figures, that goes beyond the occasional oppositionality seen as a normal behavior in individuals at a certain age, e.g., during child and adolescent development."[PMID:27035043]
**Comment**: Age-inappropriate oppositional behavior is the characteristic feature of oppositional defiant disorder (ODD), which is
diagnosed broadly on the basis of frequent and persistent angry or irritable mood, argumentativeness/defiance, and vindictiveness. Diagnostic criteria for ODD were first introduced in Diagnostic and Statistical Manual of Mental Disorders (DSM)-V also include criteria about angry/irritable mood and vindictiveness, a stipulation that the disturbance in behavior is associated with distress in the individual or others in his or her immediate social context, and a requirement that the behavior does not occur exclusively during the course of a psychotic, substance use, depressive, or bipolar disorder.  These features should be persistent, with duration of at least 6 months. This HPO term intends to describe the phenotypic feature of excessive oppositional behavior rather than the diagnosis of ODD, and these additional criteria are not needed to apply the HPO term to describe an individual.

**Synonyms**
Oppositional defiant disorder - related synonym
ODD related synonym, abbreviation
Excessive oppositional behavior - exact
Developmentally inappropriate oppositional behavior - exact

**Comment by pnrobinson** [comment]
added

**Comment by stefaniadv** [comment]
Thank you very much for your help!
Best wishes
Stefania
